.travis.yml

```diff
@@ -18,6 +18,7 @@ matrix:
 before_install:
     - export COMPOSER_GLOBAL_HOME_DIR=`composer config -g home`
     - sed s\@TRAVIS_BUILD_DIR@$TRAVIS_BUILD_DIR@g test/travis-composer.json > ${COMPOSER_GLOBAL_HOME_DIR}/composer.json
+    - curl -LO https://www.phing.info/get/phing-latest.phar
```

Comment:
this can break when phing will release new major version that includes bc breakers